pypit/arproc.py

```diff
@@ -930,6 +930,20 @@ def reduce_frame(slf, sciframe, scidx, fitsdict, det, standard=False):
         flex_dict = arwave.flexure_obj(slf, det)
         arqa.flexure(slf, det, flex_dict)
 
+    # Helio-centric
+    if settings.argflag['reduce']['calibrate']['refframe'] == 'heliocentric':
```

Comment:
Since implementing the barycentric correction is now almost as easy as the heliocentric correction, I propose we allow both methods here. If you agree, here are a few changes:

if settings.argflag['reduce']['calibrate']['refframe'] in ['heliocentric', 'barycentric']:

then delete lines 937-938